pkg/gatherers/clusterconfig/openshift_logging_test.go

```diff
@@ -0,0 +1,58 @@
+package clusterconfig
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	dynamicfake "k8s.io/client-go/dynamic/fake"
+)
+
+func Test_gatherOpenshiftLogging(t *testing.T) {
```

Comment:
We usually call the tests `Test_Gather....` AFAIK

Reply:
typo